engine/src/main.cpp

```diff
@@ -273,9 +273,11 @@ int main( int argc, char *argv[] )
 
     VegaStrikeLogging::VegaStrikeLogger::InitLoggingPart1();
 
-#ifdef WIN32
-    VSFileSystem::InitHomeDirectory();
-#endif
+    // stephengtuggy 2021-09-10 Let's try initializing the directories in the same order
+    // on Windows as on other platforms, and see what happens
+// #ifdef WIN32
+//     VSFileSystem::InitHomeDirectory();
+// #endif
```

Comment:
:+1:

Reply:
As a nice bonus, this change seems to have enabled PWCU to start on Windows without crashing.